manifest

```diff
@@ -185,6 +186,7 @@ export SERVICES="\
 	haveged \
 	systemd-timesyncd \
 	sshd \
+	neo-controller \
```

Comment:
Does this service run on all hardware? Could that cause problems?

Reply:
The script the service executes checks for known Aya neo devices first and exits if it doesn't exist.

https://github.com/ShadowBlip/aya-neo-fixes/blob/c5b7293813415b2cb6245079597060efb7b65648/neo-controller.py#L27